conformance/utils/suite/experimental_reports.go

```diff
@@ -182,7 +182,7 @@ func isTestExtended(profile ConformanceProfile, test ConformanceTest) bool {
 	for _, supportedFeature := range test.Features {
 		// if ANY of the features needed for the test are extended features,
 		// then we consider the entire test extended level support.
-		if profile.ExtendedFeatures.Has(supportedFeature) {
```

Comment:
Why this change? Isn't it equivalent?

Reply:
The checks aren't equivalent because for some reason I can't say I really understand `GatewayExtendedFeatures` also includes all `GatewayCoreFeatures`: https://github.com/kubernetes-sigs/gateway-api/blob/5fd8bec3bedc7c325bf2227da48d041a735b9b98/conformance/utils/suite/features.go#L64